onnxruntime/core/providers/rknpu/onnx_converter.h

```diff
@@ -0,0 +1,239 @@
+// Copyright 2020 rock-chips.com Inc.
+
+#pragma once
+
+#include <onnx/onnx_pb.h>
+
+#include <map>
+#include <memory>
+#include <utility>
+#include <vector>
+#include <unordered_map>
+#include <string>
+
+#include "shaper.h"
+
+#include "rknpu/rknpu_pub.h"
+
+namespace onnxruntime {
+namespace rknpu {
+class OnnxConverter {
```

Comment:
>lass OnnxConverter { [](start = 1, length = 20)

The class and its methods lack documentation.

Reply:
already added.